RPIPs/RPIP-47.md

```diff
@@ -33,8 +33,8 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
   - Setting the previous `megapool delegate`'s expiration block to occur `delegate_upgrade_buffer` after the upgrade
 - A node operator SHALL be able to upgrade their active `megapool delegate` to the latest `megapool delegate` at any time (including after the active `megapool delegate`'s expiration block)
 - After the active `megapool delegate`'s expiration block:
-  - The active `megapool delegate` SHALL NOT be usable by the node operator
-  - The node operator SHALL NOT be able to claim any rewards
+  - Functions in the active megapool delegate SHALL revert when not called by a protocol contract
```

Comment:
I don't really understand what this change is about, seems to be quite different.

Reply:
It was a clarification change. It wasn't clear what the extent of 'not usable' was. So it was made more explicit: 'NO can't do anything, everything reverts unless called through another protocol contract.'